src/Maths/Silk.NET.Maths/Simd/Simd128.GreaterThanOrEqual.cs

```diff
@@ -0,0 +1,176 @@
+                {
+#if AdvSIMD
+                    if (AdvSimd.IsSupported)
+                    {
+                        return AdvSimd.CompareGreaterThanOrEqual(left.AsInt16(), right.AsInt16()).As<short, T>();
+                    }
+#endif
+                }
+        
+                return UInt32(left, right);
+            }            
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> UInt32(Vector128<T> left, Vector128<T> right)
+            {
+                if (typeof(T) == typeof(uint))
+                {
+#if AdvSIMD
+                    if (AdvSimd.IsSupported)
+                    {
+                        return AdvSimd.CompareGreaterThanOrEqual(left.AsUInt32(), right.AsUInt32()).As<uint, T>();
+                    }
+#endif
+                }
+        
+                return Int32(left, right);
+            }            
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> Int32(Vector128<T> left, Vector128<T> right)
+            {
+                if (typeof(T) == typeof(int))
+                {
+#if AdvSIMD
+                    if (AdvSimd.IsSupported)
+                    {
+                        return AdvSimd.CompareGreaterThanOrEqual(left.AsInt32(), right.AsInt32()).As<int, T>();
+                    }
+#endif
+                }
+        
+                return Single(left, right);
+            }                     
+   
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> Single(Vector128<T> left, Vector128<T> right)
+            {
+                if (typeof(T) == typeof(float))
+                {
+#if AdvSIMD
+                    if (AdvSimd.IsSupported)
+                    {
+                        return AdvSimd.CompareGreaterThanOrEqual(left.AsSingle(), right.AsSingle()).As<float, T>();
+                    }
+#endif
+#if SSE
+                    if (Sse2.IsSupported)
+                    {
+                        return Sse2.CompareGreaterThanOrEqual(left.AsSingle(), right.AsSingle()).As<float, T>();
+                    }
+#endif
+                }
+        
+                return Double(left, right);
+            }            
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> Double(Vector128<T> left, Vector128<T> right)
+            {
+                if (typeof(T) == typeof(double))
+                {
+#if SSE
+                    if (Sse2.IsSupported)
+                    {
+                        return Sse2.CompareGreaterThanOrEqual(left.AsDouble(), right.AsDouble()).As<double, T>();
+                    }
+#endif
+                }
+        
+                return Integer(left, right);
+            }
+            
+            [MethodImpl(Scalar.MaxOpt)]
+            static Vector128<T> Integer(Vector128<T> left, Vector128<T> right)
+            {
+                if (Simd128<T>.IsHardwareAccelerated)
+                    return Or(GreaterThan(left, right), Equal(left, right));
```

Comment:
Is this better then Not(LessThan(left, right))?

Reply:
Indeed, it's probably not. I keep forgetting that `a > b = b < a`